domain/src/main/java/mihon/domain/extensionrepo/interactor/CreateExtensionRepo.kt

```diff
@@ -14,11 +14,12 @@ class CreateExtensionRepo(
     private val repoRegex = """^https://.*/index\.min\.json$""".toRegex()
 
     suspend fun await(repoUrl: String): Result {
-        if (!repoUrl.matches(repoRegex)) {
+        val cleanedRepoUrl = repoUrl.trim().lowercase()
```

Comment:
Lowercasing a url seems like a bad idea

Reply:
> Lowercasing a url seems like a bad idea

How about at least lowercasing the protocol bit?

Reply:
https://www.rfc-editor.org/rfc/rfc3986

So, the scheme and host are case-insensitive.
The rest of the URL is case-sensitive.

Reply:
So basically, lowercasing the scheme should be fine. I'll adjust it in a moment.

Reply:
Fixed by case-insensitively replacing the first match for the `^https` regex.